test/functional/buffered_attributes.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license
+ * Copyright 2018 The Incremental DOM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {applyAttrs, attr, close, open, patch} from '../../index';
+const {expect} = chai;
+
+describe('buffered attributes', () => {
+  let container: HTMLElement;
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
+  });
+
+  it('should add attributes to the current element', () => {
+    patch(container, () => {
+      open('div');
+      // Attrs below will still apply to the current pointer, even though
+      // another element was opened/closed in the mean time.
+      open('span');
+      close();
+
+      attr('nameOne', 'valueOne');
```

Comment:
how do statics work?

Reply:
Changed the `applyStatics` and added tests.